funcx_sdk/funcx/sdk/client.py

```diff
@@ -57,7 +53,6 @@ def __init__(
         search_authorizer=None,
         openid_authorizer=None,
         funcx_service_address=None,
-        check_endpoint_version=False,
         asynchronous=False,
```

Comment:
Looks like there's at least one (non-fatal) usage still around in `def start_endpoint()`.

Reply:
I removed that, and then I reverted it because I was thinking that maybe we wanted it to stick around for a version. Now I'm looking and I'm going to remove it again!

I don't see a very strong argument in either direction, and you're noting it, so I'm going to remove the usage.